### DISCUSSION TITLE
UI: see all faces of one person (close-up)

### DISCUSSION THREAD

**Discussion by Tirallala** [photoprism/photoprism#2977] [Ideas]
When I click on a person, I see all pictures which contain faces assigned to that person. I'd like to have the possibility not to see the picture, but the faces (close-up of the head). The purpose of this is to check if the faces have been assigned to the proper person - not one by one, but multiple faces at the same time.

I know there is the possibility to move through the faces of one person one by one when I start from the people tab while editing a picture. But this doesn't help in checking multiple faces of one person.

Example: I'd like to check if the faces assigned to person luis are really luis. So I click on the person and get something like the screenshot. This dosn't help in checking for mis-assigned faces.

Could there maybe be an additional view to switch between full picture and close-up?
![person](https://user-images.githubusercontent.com/106486840/206293802-998bdea2-a8bb-4d6c-aaed-51a661dea507.PNG)

**Comment by graciousgrey** [comment]
Sounds similar to: https://github.com/photoprism/photoprism/issues/1601
